subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/generator/KotlinDslFileContentGenerator.groovy

```diff
@@ -55,7 +55,7 @@ class KotlinDslFileContentGenerator extends FileContentGenerator {
             minHeapSize = testRunnerMemory
             maxHeapSize = testRunnerMemory
             maxParallelForks = ${config.maxParallelForks}
-            setForkEvery(testForkEvery.toLong())
+            forkEvery = testForkEvery.toLong()
```

Comment:
Is this auto-unboxing to a long? I think toLong returns a `Long`, so this is calling `setForkEvery(Long)`.

Reply:
This is auto-unboxing. In Kotlin only the setter strictly matching is usable via property syntax.

Reply:
But in this case I had to revert the change because this is generating performance test templates and they need to work with previous Gradle versions :roll_eyes: